getting_started_modules.md

```diff
@@ -0,0 +1,167 @@
+## Getting Started
+
+### Goal of This Guide
+
+> :warning: Read the section `Description - what & why` in [README.md](/README.md) to get a quick introduction to what this repo is.
+
+The template you found this README in is specifically built to make it as easy and quick as possible to make terraform modules, and then test them inside the [vagrant-hashistack](https://app.vagrantup.com/fredrikhgrelland/boxes/hashistack) box. 
+
+This guide aims to show you how to use this template. The steps we are going to walk through are as follows: TODO:UPDATE THIS LIST
+
+- Building a docker image
+- Creating a nomad job that uses this image
+- Creating the terraform module
+- Making the nomad job more dynamic
+
+
+### The development process
+
+#### 1. Building Docker Image
+
+> :warning: This section is only relevant if you want to build your own docker image. Skip this if you are new.
+
+Most of the terraform modules will deploy one or more docker-containers to Nomad. If you want to create your own docker-image, put the [`Dockerfile`](https://docs.docker.com/engine/reference/builder/) under [docker/](/docker/).
+
+#### 2. Deploying Container With Nomad
+At this point you should have a service that will run when you start the docker container. TODO:ADD LINK TO SIMPLE CONTAINER TO USE FOR GUIDE Either you've made a container yourself, or you are using some other container. The next step is then to deploy this container to our hashistack ecosystem. Nomad is running inside our virtual machine, and is used to deploy containers, and register them into Consul. It also has a tight integration with vault that we will use later. 
```

Comment:
```suggestion
At this point you should have a service that will run when you start the docker container. TODO:ADD LINK TO SIMPLE CONTAINER TO USE FOR GUIDE Either you've made a container yourself, or you are using some other container. The next step is then to deploy this container to our hashistack ecosystem. Nomad is running inside a virtual machine and is used to deploy containers and register them into Consul. It also has tight integration with vault that we will use later. 
```

Reply:
I honestly like `our` better, it makes it clear we are talking about the virtual machine that has been started earlier.

WDYT?

Reply:
it's ok for me.